app/extensions.js

```diff
@@ -137,19 +151,52 @@ let generateBraveManifest = () => {
     cspDirectives['style-src'] = '\'self\' \'unsafe-inline\' http://' + devServer
   }
 
+  baseManifest.content_security_policy = concatCSP(cspDirectives)
+
+  return baseManifest
+}
+
+// Returns the Chromium extension manifest for the torrentExtension
+// The torrentExtension handles magnet: URLs
+// Analagous to the PDFJS extension, it shows a special UI for that type of resource
+let generateTorrentManifest = () => {
+  let cspDirectives = {
+    'default-src': '\'self\'',
+    'form-action': '\'none\'',
+    'referrer': 'no-referrer',
+    'style-src': '\'self\' \'unsafe-inline\'',
```

Comment:
I don't think we use inline `<style>` tags, so we can remove `unsafe-inline` here.